composer.json

```diff
@@ -27,14 +27,14 @@
     "extra": {
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0"
+        "php": "^7.4 || ~8.0.0 || ~8.1.0 || ~8.2.0"
```

Comment:
Does this mean we need another patch next month to remove support for 7.4?

Reply:
There's some ^7.x specific tests here that should also be dropped - it'd be better off in a separate patch.

That said, what's the policy for dropping dead versions of PHP? Is it "As soon as they die", or when the lib wants to use features not available in EOL'd version?

Reply:
The question is - should all repositories be touched twice in a short time for these version updates? Has it already been clarified what the best variant would be here?

Reply:
I don't think this has been clarified… Taken to Slack… 👍